packages/mobile/src/functions/handleFormSubmit.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {ToastAndroid} from 'react-native';
+import postUser from '../api/PostUser';
+
+const handleFormSubmit = async (values: any, actions: any) => {
+  const user = {
+    email: values.email,
+    password: values.password,
+  };
+  actions.resetForm();
+  try {
+    const res = await postUser(user);
+    console.log(res);
+    ToastAndroid.showWithGravity(
```

Comment:
A co się stanie jak justyna spróbuje uruchomić to na iphone? sprawdzaj jaki to system zanim odpalisz coś co istnieje tylko dla jednego systemu

Reply:
Platform.OS === 'ios' ? cos : cosinnego